Makefrag

```diff
@@ -25,7 +25,11 @@ build_dir=$(sim_dir)/generated-src
 
 CHISEL_ARGS ?=
 
+ifneq ($(PROJECT),example)
+long_name=$(PROJECT).$(CONFIG)
+else
 long_name=$(PROJECT).$(MODEL).$(CONFIG)
+endif
```

Comment:
Hmm why do we not use `$(PROJECT).$(MODEL).$(CONFIG)` for non-example projects? Also `ifneq then else` hurts my brain.

Reply:
hwacha and rocket-chip have different long_names but this works for now.

Reply:
ok